tutorial/src/00-setup.md

```diff
@@ -11,12 +11,13 @@ npm i -d decentraland-ecs
 npx dcl init
 ```
 
-TODO do we need the decentraland-ecs or does that come from dcl init?
+// TODO do we need the decentraland-ecs or does that come from dcl init?
+//No need, it comes from dcl init
 
 Then run:
 
 ```
-npm run start
+dcl start
```

Comment:
Hmm we either need to use a `-g` install or use npm (or npx) commands.  This won't work if the follow the instructions above.

I personally prefer the local install - it lets our package.json define the version and is a little easier to manage once we have other dependencies in the mix.

But I see the docs prefer a global install.  So thinking switching to that may be best.

Reply:
I hear your point, but we'd want to keep this super simple for people that are barely familiar with using the command line. Running a single command from the `dcl` tool seems to me as simple as it can get.